jetbrains-core/tst/software/aws/toolkits/jetbrains/core/explorer/AwsExplorerNodeProcessorTest.kt

```diff
@@ -93,7 +93,7 @@ class AwsExplorerNodeProcessorTest {
             }
         }
 
-        countDownLatch.await(1, TimeUnit.SECONDS)
+        countDownLatch.await(10, TimeUnit.SECONDS)
```

Comment:
Woah, what step update required this bump?

Reply:
not sure why the tests were timing out with 1s, but the test itself does not take that long